tasks/get_objects_for_app.yml

```diff
@@ -18,6 +18,7 @@
     deployments: "{{ templates | map('regex_search', '.*/dc.*\\.yml\\.j2$') | select('string') | list }}"
     endpoints: "{{ templates | map('regex_search', '.*/ep.*\\.yml\\.j2$') | select('string') | list }}"
     services: "{{ templates | map('regex_search', '.*/svc.*\\.yml\\.j2$') | select('string') | list }}"
+    static_services: "{{ templates | map('regex_search', '.*/static-svc.*\\.yml\\.j2$') | select('string') | list }}"
```

Comment:
why do you need static-svc to coexist with svc? My understanding is that now all services will be removed from blue/green actions.

Reply:
sorry sorry I read your PR descriptyion after the code and now I undersatand. Smart :clap: